domains/authorization/Authorization.API/API.UnitTests/API.UnitTests/[Commands]/GetConsentForUserQueryHandlerTests.cs

```diff
@@ -0,0 +1,138 @@
+using API.Authorization._Features_;
+using API.Data;
+using API.Models;
+using API.Repository;
+using API.UnitTests.Repository;
+using API.ValueObjects;
+using FluentAssertions;
+using NSubstitute;
+
+namespace API.UnitTests._Commands_;
+
+public class GetConsentForUserQueryHandlerTests
```

Comment:
What about the case where a new version of Terms exists, what is the expected behavior in that case?

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1279/commits/45fba051f7cff543c28fbc209992b389f2ee64ba